core/trino-main/src/main/java/io/trino/operator/aggregation/WindowAccumulator.java

```diff
@@ -24,7 +24,7 @@ public interface WindowAccumulator
 
     void addInput(WindowIndex index, int startPosition, int endPosition);
 
-    void removeInput(WindowIndex index, int startPosition, int endPosition);
+    boolean removeInput(WindowIndex index, int startPosition, int endPosition);
```

Comment:
Let’s add a JavaDoc comment on the method that explains the contract for what the boolean return indicates

Reply:
Done